docs/releases/202409.mdx

```diff
@@ -0,0 +1,18 @@
+---
+id: 202409
+title: September 2024
+displayed_sidebar: releaseSideBar
+---
+
+### September 5 2024
+
+**Features**
+- **Agent**: A new agent type is available that supports more detailed file inspection and manipulation prior to uploading to the Ganymede cloud
```

Comment:
This is probably OK, since I'll prob add a section for the Agent with some more details / examples on usage + link it.

My description would be:
```A new Agent type, similar to the 'Watch for files locally, then run flow' Agent type, is now available.
- The key difference is that flow execution is triggered by user-defined code rather than by matching each specified glob pattern.  This enables users to develop Agents in which Flow execution is determined by file content, not just the file name.
- The notebook template for the new Agent also lets users simulate files, improving Agent testability prior to installation.
```

Reply:
Sounds good, I'll keep an eye out for that PR